resources/views/layouts/app.blade.php

```diff
@@ -307,5 +309,21 @@
         @yield('javascript')
         @action('javascript', $__env->yieldContent('javascripts'))
     </script>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
```

Comment:
We need to keep all the code in local as many of the bank doesn't have internet.